src/chromium/adding-third-party-crates/resolving-problems/build-scripts-which-take-arbitrary-actions.md

```diff
@@ -1,7 +1,7 @@
 # Build scripts which build C++ or take arbitrary actions
 
-Some crates use the [`cc`][2] crate to build and link C/C++ libraries.
-Other crates parse C/C++ using [`bindgen`][3] within their build scripts.
+Some crates use the [`cc`][1] crate to build and link C/C++ libraries.
+Other crates parse C/C++ using [`bindgen`][2] within their build scripts.
```

Comment:
Just as a note, I've found the [shortcut reference link](https://spec.commonmark.org/0.30/#shortcut-reference-link) syntax very useful: just `` [`cc`] `` in this case and then a corresponding

```markdown
[`cc`]: ...
```

definition further down.